pkg/datapath/connector/veth.go

```diff
@@ -32,7 +32,7 @@ func SetupVethRemoteNs(netNs ns.NetNS, srcIfName, dstIfName string) error {
 // fields such as mac, NodeMac, ifIndex and ifName. Returns a pointer for the created
 // veth, a pointer for the temporary link, the name of the temporary link and error if
 // something fails.
-func SetupVeth(id string, mtu, groIPv6MaxSize, gsoIPv6MaxSize, groIPv4MaxSize, gsoIPv4MaxSize int, ep *models.EndpointChangeRequest) (*netlink.Veth, netlink.Link, string, error) {
+func SetupVeth(id string, mtu, groIPv6MaxSize, gsoIPv6MaxSize, groIPv4MaxSize, gsoIPv4MaxSize int, ep *models.EndpointChangeRequest, sysctlDisable func(name, val string) error) (*netlink.Veth, netlink.Link, string, error) {
```

Comment:
Why is the callback still necessary? Cant we pass a `sysctl.Sysctl` to `connector.SetupVeth`?